### PR TITLE
Changing build, build def to ensure Nuget files are copied to output, fixing nuget license and project URLs

### PR DESCRIPTION
**Pull request by AlexBulankou**
Changing build, build def to ensure Nuget files are copied to output, fixing nuget license and project URLs

### CODE REVIEW

File: JavaScript/Microsoft.ApplicationInsights.JavaScript.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 2013
-VisualStudioVersion = 12.0.40418.0
+VisualStudioVersion = 12.0.31101.0
```
Comment (ameyagholkar) [inline JavaScript/Microsoft.ApplicationInsights.JavaScript.sln]: We are going back in version numbers? Is this intentional?

**Comment by MaxShehovtsov** [comment]
MWAH

File: JavaScript/Microsoft.ApplicationInsights.JavaScript.sln (hunk 1 above)
Comment (AlexBulankou) [inline JavaScript/Microsoft.ApplicationInsights.JavaScript.sln]: Not intentional, will fix

**Comment by ameyagholkar** [comment]
:sheep: it

**Pull request merged**